relayer/band/types/signing.go

```diff
@@ -25,10 +25,11 @@ func NewEVMSignature(
 
 // Signing contains information of a requested message and group signature.
 type Signing struct {
-	ID           uint64           `json:"id"`
-	Message      cmbytes.HexBytes `json:"message"`
-	EVMSignature *EVMSignature    `json:"evm_signature"`
-	Status       string           `json:"signing_status"`
+	ID                  uint64                 `json:"id"`
+	Message             cmbytes.HexBytes       `json:"message"`
+	EVMSignature        *EVMSignature          `json:"evm_signature"`
+	SigningStatus       tsstypes.SigningStatus `json:"-"`
+	SigningStatusString string                 `json:"signing_status"`
```

Comment:
the purpose of this field is just for json marshal and got string, right?

if yes, should we do custom marshal instead? (example code from gpt below). anyway, I'm ok for both way.
```
// Custom MarshalJSON to include SigningStatusString dynamically
func (s *Signing) MarshalJSON() ([]byte, error) {
	type Alias Signing // Avoids infinite recursion
	return json.Marshal(&struct {
		*Alias
		SigningStatusString string `json:"signing_status"`
	}{
		Alias:               (*Alias)(s),
		SigningStatusString: tsstypes.SigningStatus_name[int32(s.SigningStatus)],
	})
}
```